packages/gatsby/CHANGELOG.md

```diff
@@ -7635,7 +7635,7 @@ Our first beta!!! 🎉
 ### Added
 
 - Allow for gatsby-remark-smartypants options #1166 @mitchejj
-- New design (for gatsbyjs.org) + new home page #1170 @kyleamathews
+- New design (for gatsbyjs.com) + new home page #1170 @kyleamathews
```

Comment:
I'd maybe remove the changes to the CHANGELOG here since this is a historical record